src/Microsoft.OData.Edm/ExtensionMethods/EdmTypeSemantics.cs

```diff
@@ -1216,18 +1221,39 @@ public static EdmPrimitiveTypeKind PrimitiveKind(this IEdmTypeReference type)
         /// <param name="potentialBaseType">The potential base type of the type being tested.</param>
         /// <returns>True if and only if the type inherits from the potential base type.</returns>
         public static bool InheritsFrom(this IEdmStructuredType type, IEdmStructuredType potentialBaseType)

```

Comment:
>InheritsFrom [](start = 27, length = 12)

I 'd like to see the perf telemetry about this change.

It's easy to test, create an edm model with a big inheritance type true,

A => B => C => D => ....

Call this method multiple time and record the time duration.